src/app/add-clients/helperFunctions.tsx

```diff
@@ -0,0 +1,254 @@
+import React from "react";
+import { Database } from "@/database_types_file";
+import { booleanGroup } from "@/components/DataInput/inputHandlerFactories";
+import { SelectChangeEvent } from "@mui/material";
+import { InsertSchema } from "@/supabase";
+import supabase from "@/supabase";
+
+type PersonType = Database["public"]["Enums"]["gender"];
```

Comment:
rename to Gender?

Reply:
We're using PersonType to match what's on the database. We can change it to "gender" on the database but I think that wouldn't be the most suitable naming since there are options like "boy", "girl", "adult" and "child"